app-admin/testdisk/testdisk-7.1~git.recipe

```diff
@@ -96,10 +107,9 @@ BUILD()
 {
 	export CXXFLAGS="-std=c++11"
 	autoreconf -fi
-	runConfigure --omit-dirs "binDir" ./configure \
-		--bindir=$prefix/bin \
+	runConfigure --omit-dirs binDir ./configure --bindir="$commandBinDir" \
 		--disable-stack-protector
-	make $jobArgs
+	make $jobArgs LDFLAGS="-lintl"
```

Comment:
Why is gettext being forcefully linked here? Also, for adding library to the linking process, please use the `LIBS` variable instead (ie. `LIBS="-lintl"`), as it will appends the said library to the end of linker flags, so that flags used by the build system are applied (ie. `--as-needed` requires libs to be passed after the switch)

The environment variable should be declared prior to `runConfigure`, so the build system can pick it up and process it accordingly.

Reply:
I had an error about a symbol that could not be resolved, and I figured out that I could get rid of this error by just adding `-lintl`.
But there might very well be a better fix. (I'll try to investigate.)